modin/backends/pandas/parsers.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import pandas
+from modin.engines.ray.generic.io import file_open
+from modin.engines.ray.pandas_on_ray.io.io import _split_result_for_readers
+from io import BytesIO
+
+
+class PandasCSVParser:
+    @classmethod
+    def parse(cls, **kwargs):
```

Comment:
This should return a callable.

Reply:
I thought it was supposed to do the parsing?

Reply:
I suppose it should be fine.

Reply:
I was thinking a little differently, but your way makes more sense.